Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM python:3.7-buster as base
+RUN pip install poetry
+COPY poetry.toml poetry.lock pyproject.toml /
+
+FROM base as production
+RUN poetry install -n
+COPY /todo_app /todo_app
+EXPOSE 8000
+ENTRYPOINT poetry run gunicorn --bind 0.0.0.0 "todo_app.app:create_app()"
+
+FROM base as development
+RUN poetry install -n --without prod
```

Comment:
I might be misunderstanding, but don't you want to run:
1. `poetry install -n` for development
2. `poetry install -n --without dev` for production?

Reply:
Because, as far as I can tell from the `pyproject.toml` file, you have explicitly set dependencies for dev (which are not used by prod), but not for prod